src/vector/platform/ElectronPlatform.js

```diff
@@ -213,7 +214,7 @@ export default class ElectronPlatform extends VectorBasePlatform {
         ipcRenderer.on('update-downloaded', this.onUpdateDownloaded.bind(this));
 
         ipcRenderer.on('preferences', () => {
-            dis.dispatch({ action: 'view_user_settings' });
+            dis.fire(Action.ViewUserSettings);
```

Comment:
There are still a few dispatcher changes appearing here, is it correct?

Reply:
Sorry, I didn't merge develop in. It should be happier now.